Formula/volta.rb

```diff
@@ -0,0 +1,20 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta/releases/download/v0.7.2/volta-0.7.2-macos.tar.gz"
```

Comment:
Our policy is to build from source: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
I've updated the pull-request to build from source.